collection-hooks.js

```diff
@@ -297,9 +267,3 @@ if (typeof Mongo !== 'undefined') {
 } else {
   CollectionHooks.wrapCollection(Meteor, Meteor)
 }
-
-// Make the above available for packages with hooks that want to determine
-// whether they are running inside a publish function or not.
-CollectionHooks.isWithinPublish = function isWithinPublish () {
```

Comment:
This was falsely changed. This function should only be defined on the server - and it's already defined in the `server.js` file.